store/todo.js

```diff
@@ -14,22 +23,24 @@ export const state = () => ({
         completedIn: task.completedIn
       })
     },
-    removeTask(state, taskId) {
-      const task = state.getters.getTaskById(taskId)
-      state.todoList.splice(state.todoList.indexOf(task), 1)
+    incrementTaskId(state) {
+      state.taskId++
```

Comment:
Rather than incrementing a taskId yourself, can we use universal ids? We can use [uuidv4](https://www.npmjs.com/package/uuidv4) library to do so.

Reply:
added uuidv4 library